tests/test_keras.py

```diff
@@ -7,9 +7,6 @@
 from conversion_imagenet import TestModels
 
 def get_test_table():
-    if six.PY3:
```

Comment:
@BlaiseRitchie , please don't delete these lines of code since they control different model sets to be tested on Travis or your local environment. You could simply add the elutestnet in the following and tests/conversion_imagenet.py (around line 830)